tests/storages_tests/rdb_tests/test_storage.py

```diff
@@ -119,135 +110,6 @@ def test_set_default_engine_kwargs_for_mysql_with_other_rdb():
     assert "pool_pre_ping" not in engine_kwargs
 
 
```

Comment:
comment: Logic tests for RDB storage are merged into `tests/storage_tests/test_storages.py`.